complete/src/main/java/com/example/restservice/model/CreateQueueRequest.java

```diff
@@ -1,11 +1,17 @@
 package com.example.restservice.model;
 
+
+import com.fasterxml.jackson.annotation.JsonProperty;
 import lombok.Data;
 
 @Data
 public class CreateQueueRequest {
   String queueName;
 
 
+  @JsonProperty("isPasswordProtected")
```

Comment:
Do we need to explicitly set this?

Reply:
I forget to ask this.!
If I use primitive data type boolean, jackson will remove "is" keyword while parsing. So the field will not be set correctly. I was using primitives in model class earlier, that's why explicitly provided.

But later noticed you have used wrapper Boolean data type in `JoinQueueRequest` . So changed `isPasswordProtected ` to Boolean data type. So annotation is not required now. I will remove it.

Just for my understanding, is there a convention to use wrapper data type on model. I can see Boolean is used for `JoinQueueRequest` class field `notifyable` , but primitive long is used in `UserStatusResponse` field `aheadCount`.

Reply:
It's generally a good idea to use boxed types: https://stackoverflow.com/questions/51642979/boxed-vs-primitive-type-as-entity-id

It makes sense, null is better than accidentally setting something to 0. Can you also change notifyable to Boolean?

Reply:
Done. Changed all primitives in model class.